awx/main/models/ha.py

```diff
@@ -161,6 +161,24 @@ def consumed_capacity(self):
     def remaining_capacity(self):
         return self.capacity - self.consumed_capacity
 
+    @staticmethod
+    def get_remaining_capacity(instances={}, jobs=[]):
```

Comment:
Name isn't good anymore. I don't even like that this is on the `Instance` model, because the `instances` data structure is not populated with `Instance`s. No, the `instances` values are full of `types.SimpleNamespace`s, which are informally referred to as "partial instances".

This more-or-less starts to do the same thing as `capacity_values`. That's okay by me, because our current direction should lead to deleting the `capacity_values` method.

Reply:
I agree with @jbradberry 

I stand by my point that this method should be renamed.

I retract my request to move the method, as it is marked as `@staticmethod` and operates on partial instances. A pattern for this already exists, and a refactor of that should be handled separately from your present work.

Reply:
ok will move it back, rename it, and remove the defaults

Reply:
yeah the task manager is not using `capacity_values` anymore, but the API is...you're deprecation of `committed_capacity` should help move us in that direction. `consumed_capacity` also uses it though, you should maybe think about if we can deprecate that too

Reply:
The latter, we can move the serializer to use the method you wrote, and then purge references to the old `capacity_values`, and delete it. This is assuming this goes in, and that the field deprecation is merged too.